torchvision/utils.py

```diff
@@ -300,6 +301,70 @@ def draw_segmentation_masks(
     return out.to(out_dtype)
 
 
+@torch.no_grad()
+def draw_keypoints(
+    image: torch.Tensor,
+    keypoints: torch.Tensor,
+    labels: Optional[List[str]] = None,
+    colors: Optional[Union[List[Union[str, Tuple[int, int, int]]], str, Tuple[int, int, int]]] = None,
+    radius: int = 2,
+    connect: bool = False,
+    font: Optional[str] = None,
+    font_size: int = 10
+) -> torch.Tensor:
+
+    """
+    Draws Keypoints on given RGB image.
+    The values of the input image should be uint8 between 0 and 255.
+
+    Args:
+        image (Tensor): Tensor of shape (3, H, W) and dtype uint8.
+        keypoints (Tensor): Tensor of shape (num_keypoints, K, 3) the K keypoints location for each of the N instances,
+            in the format [x, y, visibility], where visibility=0 means that the keypoint is not visible.
+        labels (List[str]): List containing the labels for each Keypoint.
+        colors (Union[List[Union[str, Tuple[int, int, int]]], str, Tuple[int, int, int]]): List containing the colors
+            or a single color for all the keypoints.
+            The colors can be represented as `str` or `Tuple[int, int, int]`.
+        radius (int): Integer denoting radius of keypoint.
+        connect (bool): If True. It connects all the visible keypints.
+        font (str): A filename containing a TrueType font. If the file is not found in this filename, the loader may
+            also search in other directories, such as the `fonts/` directory on Windows or `/Library/Fonts/`,
+            `/System/Library/Fonts/` and `~/Library/Fonts/` on macOS.
+        font_size (int): The requested font size in points.
```

Comment:
If we don't provide labels, we don't need to provide font either

Reply:
Removing labels as suggested.